static/src/javascripts/boot.js

```diff
@@ -171,10 +171,6 @@ const go = () => {
 		const fetchCommercial = () => {
 			const noop = () => Promise.resolve({ bootCommercial: () => {} });
 
-			if (!config.get('switches.commercial')) {
```

Comment:
Lets remember to add the new kill switch before a malicious AGI evolves from an interscroller.